### PR TITLE
Develop to truss

### PR DESCRIPTION
**Pull request by briehl**
Resolve conflicts and merge recent changes to the develop branch.

### CODE REVIEW

File: kbase-extension/static/kbase/js/widgets/function_output/rna-seq/kbaseExpressionSampleTableNew.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
+.KBaseExpressionSampleTableNew .KBaseExpressionSampleTableNew-header {
```
Comment (ialarmedalien) [inline kbase-extension/static/kbase/js/widgets/function_output/rna-seq/kbaseExpressionSampleTableNew.css:1]: oh joy... another random location for css files 😖

**Review by ialarmedalien — APPROVED** [review]
Seems reasonable.

**Comment by ialarmedalien** [comment]
Did you notice that the same unit test failed in both runs? The code in the module itself does not seem to have changed so presumably it’s something in the deps (bootstrap? or in the css?). Looks like an easy minor fix, just edit line 205 in kbaseNarrativeDataList-spec.js

**Review by ialarmedalien — CHANGES_REQUESTED** [review]
Looks good - just fix the test that fails in both unit test runs.

**Review by ialarmedalien — APPROVED** [review]
I fixed the datalist test.

File: kbase-extension/static/kbase/js/widgets/narrative_core/upload/stagingAreaViewer.js [hunk 2]
```diff
@@ -874,9 +875,15 @@ define([
             $.fn.DataTable.ext.search.pop();
 
             $.fn.DataTable.ext.search.push((_settings, _data, dataIndex) => {
+                if (_settings.nTable.id !== tableId) {
```
Comment (briehl) [inline kbase-extension/static/kbase/js/widgets/narrative_core/upload/stagingAreaViewer.js:878]: Make sure we only apply this to the staging area datatable. https://datatables.net/manual/plug-ins/search

**Pull request merged**